3 - PB/Documentazione esterna/Piano di Qualifica/Piano di Qualifica.typ

```diff
@@ -433,19 +433,19 @@ Per questo motivo, i dati utili al corretto calcolo delle metriche sono disponib
 #figure(
```

Comment:
nessun problema, solo la legenda di PPV, PAC e PEV viene visualizzata male con la dicitura PPV che esce dal riquadro

Reply:
va bene, approvo e poi andrà poi risolto in futuro